src/gstvimbasrc.c

```diff
@@ -377,21 +377,21 @@ static void gst_vimbasrc_class_init(GstVimbaSrcClass *klass)
         g_param_spec_int(
             "offsetx",
             "OffsetX feature setting",
-            "Horizontal offset from the origin to the region of interest (in pixels).",
+            "Horizontal offset from the origin to the region of interest (in pixels). If no explicit value is passed the RoI will be centered in the sensor.",
             0,
```

Comment:
Extend range to allow `-1` as new magic number for centering along an axis.

```suggestion
            -1,
```